python/ray/experimental/workflow/common.py

```diff
@@ -22,6 +23,19 @@
     from ray.experimental.workflow.storage import Storage
 
 
+@unique
+class WorkflowStatus(str, Enum):
+    # There is at least a remote task running in ray cluster
+    RUNNING = "RUNNING"
+    # It got canceled and can't be resumed later
+    CANCELED = "CANCELED"
+    # The step is finished. For virtual actor, it means that all
+    # writing steps has been finished
+    FINISHED = "FINISHED"
+    # There is an error during execution. It can be resumed
+    FAILED = "FAILED"
```

Comment:
How about calling this `RESUMABLE` instead then? Otherwise, it's ambiguous if a workflow that raised an exception is finished or failed.